inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -512,8 +511,7 @@ class Core::Impl : public ICore {
                 res = LoadNetworkImpl(cnnNetwork, plugin, parsed._config, nullptr, hash, modelPath);
             }
         } else {
-            auto cnnNetwork = ReadNetwork(modelPath, std::string());
-            res = LoadNetworkImpl(cnnNetwork, plugin, parsed._config, nullptr, {}, modelPath);
+            res = plugin.LoadNetwork(modelPath, parsed._config);
```

Comment:
@ilya-lavrenov 
One more question here: should we keep original 2-line implementation if cache is not enabled at all? E.g. add "else if (cacheManager) { ... call plugin...} else { Read + Load here}
Plugin may not know if CACHE_DIR is enabled or not

Reply:
I think we need to distinguish these cases. You are right, thanks!

Reply:
Done